src/js/handle-filters-click.js

```diff
@@ -16,17 +16,18 @@ export async function handleFiltersClick(method) {
       if (!button.classList.contains('active')) {
         filterButtons.forEach(button => button.classList.remove('active'));
         button.classList.add('active');
-      }
-      const totalPages = await method({
-        filter: button.dataset.category.replace(' ', '+'),
-        page: 1,
-      });
 
-      createPagination({
-        params: { filter: button.dataset.category.replace(' ', '+'), page: 1 },
-        totalPages,
-        method: method,
-      });
+        const totalPages = await method({
+          filter: button.dataset.category.replace(' ', '+'),
+          page: 1,
+        });
+  
+        createPagination({
+          params: { filter: button.dataset.category.replace(' ', '+'), page: 1 },
+          totalPages,
+          method: method,
+        });
+      }
```

Comment:
@tufankaracam it was inside of condition to avoid multiple clicks and not to fetch data for category which is already active

Reply:
@pavlomatchak User will go filter -> card -> exercises but on exercises user can not back same filter category. For that user can use same button like back to the category data.